### DISCUSSION TITLE
Is this a problem with wildcards?

### DISCUSSION THREAD

**Discussion by jskorlol** [spatie/laravel-permission#1896] [Q&A] [answered]
```
        Permission::create(['name' => 'admin']);
        Permission::create(['name' => 'admin.account.view']);
        Permission::create(['name' => 'admin.account.edit']);
```
I created the permission like this.

When you turn on the wildcard option

The first user with "admin" permission
All privileges of admin.* are assigned.

is this a bug?

Even though it's "admin"'s permission, not "admin.*" permission,
Taking all the rest... I think there is a difficulty in branching authority.

**Comment by PaolaRuby** [comment]
You have to give all the information, the authors cannot guess or use psychic powers to know which package version you use, php version, etc, it's the only way someone can give you an answer

**Comment by erikn69** [comment]
Try to use `'admin.index'` instead of just `'admin'`

It seems like an expected behavior, look at this test:
https://github.com/spatie/laravel-permission/blob/2512d5a22aaae0bab8d62a02e502a603776da9da/tests/WildcardHasPermissionsTest.php#L43-L47
Just with `'post'` has permission to `'post.create'`

**Comment by jbmaciel** [comment, marked as answer]
I believe this is not a bug, it is in the documentation.

![image](https://user-images.githubusercontent.com/9383831/139475350-a27a10a4-99f3-4506-a3f7-3c9949c639a0.png)
